plugin/api.go

```diff
@@ -214,6 +214,9 @@ type API interface {
 	// GetFileInfo gets a File Info for a specific fileId
 	GetFileInfo(fileId string) (*model.FileInfo, *model.AppError)
 
+	// GetFilePreview gets the bytes for a file by id.
```

Comment:
Can you add a comment documenting the minimum version here?
```suggestion
	// GetFilePreview gets the bytes for a file by id.
	//
	// Minimum server version: 5.6
```